SVN commit 849022 by lueck:

make the documentation footer translatable
translators: To benefit from the attached patch, the you have to copy the 3 new items footer-doc-comment, footer-doc-feedback and footer-doc-teamname from en.xml to your lang.xml and translate the texts
If the 3 new items don't exist in lang.xml, you get a lot of compile warnings like 'No de localization of footer-doc-comment exists; using en' and the footer is in english
CCMAIL:kde-i18n-doc@kde.org

 M  +8 -3      customization/kde-navig.xsl  
 M  +3 -0      docbook/xsl/common/de.xml  
 M  +3 -1      docbook/xsl/common/en.xml  
 M  +19 -0     docbook/xsl/common/l10n.xsl  


--- trunk/KDE/kdelibs/kdoctools/customization/kde-navig.xsl #849021:849022
@@ -205,9 +205,14 @@
         <div class="bannerBottomLeft">
           <img src="{$kde.common}bottom-left.png" style="height: 70px; margin: 0px;" alt=""/>
         </div>
-<div id="comments" style="position:relative; top: 15px; left: 1em; width: 50%; color: #282828;">
-	<p>Would you like to make a comment or contribute an update to this page?<br/>
-        Send feedback to the <a href="mailto:kde-docs@kdemail.net" style="background:transparent; color: #282828; text-decoration: underline;">KDE Docs Team</a></p>
+        <div id="comments" style="position:relative; top: 15px; left: 1em; width: 100%; color: #282828;">
+        <p>
+	<xsl:call-template name="gentext.footer-doc-comment"/>
+	<br/>
+	<xsl:call-template name="gentext.footer-doc-feedback"/>
+	<a href="mailto:kde-docs@kdemail.net" style="background:transparent; color: #282828; text-decoration: underline;">
+	<xsl:call-template name="gentext.footer-doc-teamname"/>
+	</a></p>
 	</div>
 
       </div>
--- trunk/KDE/kdelibs/kdoctools/docbook/xsl/common/de.xml #849021:849022
@@ -180,6 +180,9 @@
    <l:gentext key="hyphenation-character" text="-"/>
    <l:gentext key="hyphenation-push-character-count" text="2"/>
    <l:gentext key="hyphenation-remain-character-count" text="3"/>
++   <l:gentext key="footer-doc-comment" text="M&#246;chten Sie Kommentare und Verbesserungen zu dieser Seite beitragen?"/>
++   <l:gentext key="footer-doc-feedback" text="Senden Sie Ihre Vorschl&#228;ge an das "/>
++   <l:gentext key="footer-doc-teamname" text="KDE Dokumentationsteam"/>
 
    <l:context name="styles">
       <l:template name="person-name" text="first-last"/>
--- trunk/KDE/kdelibs/kdoctools/docbook/xsl/common/en.xml #849021:849022
@@ -180,7 +180,9 @@
    <l:gentext key="hyphenation-character" text="-"/>
    <l:gentext key="hyphenation-push-character-count" text="2"/>
    <l:gentext key="hyphenation-remain-character-count" text="2"/>
-
+   <l:gentext key="footer-doc-comment" text="Would you like to make a comment or contribute an update to this page?"/>
+   <l:gentext key="footer-doc-feedback" text="Send feedback to the "/>
+   <l:gentext key="footer-doc-teamname" text="KDE Docs Team"/>
    <l:context name="styles">
       <l:template name="person-name" text="first-last"/>
    </l:context>
--- trunk/KDE/kdelibs/kdoctools/docbook/xsl/common/l10n.xsl #849021:849022
@@ -292,6 +292,25 @@
   </xsl:call-template>
 </xsl:template>
 
+<xsl:template name="gentext.footer-doc-comment">
+  <xsl:call-template name="gentext">
+    <xsl:with-param name="key" select="'footer-doc-comment'"/>
+  </xsl:call-template>
+</xsl:template>
+
+<xsl:template name="gentext.footer-doc-feedback">
+  <xsl:call-template name="gentext">
+    <xsl:with-param name="key" select="'footer-doc-feedback'"/>
+  </xsl:call-template>
+</xsl:template>
+
+<xsl:template name="gentext.footer-doc-teamname">
+  <xsl:call-template name="gentext">
+    <xsl:with-param name="key" select="'footer-doc-teamname'"/>
+  </xsl:call-template>
+</xsl:template>
+
+
 <!-- ============================================================ -->
 
 <xsl:template name="gentext.template">